internal/pkg/log/log.go

```diff
@@ -50,16 +50,19 @@ func Init(configLogLevel string, logFile string, logInConsole bool) error {
 		return err
 	}
 
-	f, err = os.OpenFile(logPath, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0777)
-	if err != nil {
-		return err
+	fileWriter := &lumberjack.Logger{
+		Filename:   logPath,
+		MaxSize:    logFileSize, // megabytes
+		MaxBackups: 1,
+		MaxAge:     0,     //days
+		Compress:   false, // disabled by default
```

Comment:
please set compress to true, this is the reason why we need log retention: to prevent disk full by log.